tox.ini

```diff
@@ -10,5 +10,5 @@ setenv =
 deps =
     .[all]
 commands =
-    py.test --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
+    py.test {posargs} --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
```

Comment:
I did some more digging and I think `[]` is definitely not the right way. It appears to be deprecated syntax: http://lists.idyll.org/pipermail/testing-in-python/2011-January/003720.html https://code.google.com/p/pytox/issues/detail?id=15 with the curly-brackets version intended to replace it.